src/App.js

```diff
@@ -253,6 +254,7 @@ class App extends Component {
 
         return (
             <div>
+                <style>{this.props.styles}</style>
```

Comment:
Is there an elegant way to include the `<style>` tag only if `this.props.styles` is specified?

Reply:
```jsx
{this.props.styles && <style>{this.props.styles}</style>}
```

Reply:
Thx @drewwills  /  @ChristianMurphy  - added this ability in.